BAZProjectC1/Models/Strings.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Strings.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 15/09/22.
+//
+
+import Foundation
+
+enum EndpointsList:String  {
+    case movieAPI = "movieAPI"
+    case imageResorce = "imageResorce"
+    
+    var description: String {
+        switch self {
+        case .movieAPI:
+            if let endpoint = Bundle.main.object(forInfoDictionaryKey:"endpointMovieAPI") as? String{
+                return endpoint
+            }else{ return ""}
+        case .imageResorce:
+            return "https://image.tmdb.org/t/p/w500"
+        }
+    }
+}
```

Comment:
Valida el estilo de tu código. Hice algunas recomendaciones similares a las anteriores:

Algo adicional, en el caso de `movieAPI` es el retorno de un string vacío. Tal vez deberías lanzar un error explícito en este caso.

```suggestion
    var description: String {
        switch self {
        case .movieAPI:
            guard let endpoint = Bundle.main.object(forInfoDictionaryKey:"endpointMovieAPI") as? String else { return "" }
            return endpoint
        case .imageResource:
            return "https://image.tmdb.org/t/p/w500"
        }
    }
}
```